charts/xp-management/values.yaml

```diff
@@ -13,7 +13,7 @@ deployment:
     # -- Docker image repository for XP Management Service
     repository: terryyylim/xp/xp-management
```

Comment:
This value will be updated once XP PR (https://github.com/caraml-dev/xp/pull/66) is merged and new image is published. Everything else can be reviewed 🙏🏻